src/pytest/__init__.py

```diff
@@ -50,6 +50,8 @@
 from _pytest.recwarn import WarningsRecorder
 from _pytest.recwarn import warns
 from _pytest.runner import CallInfo
+from _pytest.timing import start as StartTimer
```

Comment:
no renames!

Reply:
At lines 16 and 26 there are renames already... not fair!!!

But this I can change easily...